src/test/java/model/PrizeTest.java

```diff
@@ -0,0 +1,38 @@
+package model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.*;
+
+public class PrizeTest {
+
+    @Test
+    @DisplayName("유효한 값이면 Prize 객체를 생성하는지 검증한다.")
+    public void shouldCreatePrizeWithValidValue() {
+        String validValue = "Gold";
+
+        Prize prize = new Prize(validValue);
+
+        assertThat(prize.getValue()).isEqualTo(validValue);
+    }
+
+    @Test
+    @DisplayName("값이 null이면 예외를 발생시키는지 검증한다.")
+    public void shouldThrowExceptionWhenValueIsNull() {
+        String invalidValue = null;
+
+        assertThatThrownBy(() -> new Prize(invalidValue))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("실행결과는 null이거나 공백일 수는 없습니다.");
+    }

```

Comment:
```suggestion
    @ParameterizedTest
    @NullAndEmptySource
    @DisplayName("값이 null이면 예외를 발생시키는지 검증한다.")
    public void shouldThrowExceptionWhenValueIsNull(String invalidValue) {
        assertThatThrownBy(() -> new Prize(invalidValue))
                .isInstanceOf(IllegalArgumentException.class)
                .hasMessage("실행결과는 null이거나 공백일 수는 없습니다.");
    }

```

아래의 빈문자열 테스트와 합쳐서 이렇게도 쓸 수 있을 것 같아요!

Reply:
좋은 정보 감사합니다!
보여주신 코드가 훨씬 깔끔하고 직관적인 것 같습니다.
테스트에서도 `@NullAndEmptySource`나 `@ValueSource` 같은 아노테이션을 더 활용해 보겠습니다.
감사합니다!